src/components/Releases.js

```diff
@@ -0,0 +1,132 @@
+    {
+        name: 'Theia 1.29.x (September 2022) - planned',
+        releasedate: 'September 29th, 2022',
+        releasecandidatedate:'August 25th, 2022',
+        frameworks: [
+            {
+                title: 'Eclipse GLSP',
+                url: 'https://www.eclipse.org/glsp/',
+                version: 'TBD'
+            },
+            {
+                title: 'Eclipse EMF.cloud',
+                url: 'https://www.eclipse.org/emfcloud/',
+                version: 'TBD'
+            }
+        ]
+    }
+]
+const Releases = () => (
+    <StyledResources>
+        <header className='header' role="banner">
+            <div className="row">
+                <Nav shouldRenderLogo={true} />
+                <div className="header__text-box">
+                    <h1 className="heading-primary">
+                        Releases
+                    </h1>
+                    <p>Eclipse Theia follows a monthly release cycle. You can find all releases including changelogs and breaking changes <a href="https://github.com/eclipse-theia/theia/releases">here</a></p>
+                    <br></br>
+                    <p>In addition, Theia provides a "community release" every three month with extra focus on stability and compatibility.</p>
+                    <p>The list below shows all community releases including the next one planned. In addition, you can find technologies that integrate with Theia, including the respective version that is supposed to be compatible with the community release.</p>
+                    <p>This compatibility information is provided and ensured by the respective project and therefore not in scope for the Theia project team. If you find issues with a technology, please report them at the respective project. If you are interested in getting a technology listed, please open a PR to <a href="https://github.com/eclipse-theia/theia-website">this repo</a> to be listed.</p>
+
+                </div>
+            </div>
+        </header>
+        {communityReleases.map(({ name, frameworks, releasedate, releasecandidatedate }, i) => {
+            return (
+                <section key={i} className="category row" id="frameworks">
+                    <h2 className="heading-secondary">{name}</h2>
+                    <p>Release Date: {releasedate}</p>
+                    <p>Community Release Candidate Date: {releasecandidatedate}</p>
+                    <br></br>
+                    <p><h3 className="heading-tertiary"><b>Compatible Technologies:</b></h3></p>
+                    {frameworks.map(
+                        ({ title, url, version, icon }, i) => {
+                            return (
+                                <div key={`${i}_${title}`} className="framework">
+                                            <h3 className="heading-tertiary">
+                                                <a href={url}>{title}, Version: {version}</a>
+                                            </h3>
```

Comment:
minor: should it be `h4` as "Compatible Technologies" (the title) already uses `h3` ?